### DISCUSSION TITLE
Crashing when entering single player on Vita

### DISCUSSION THREAD

**Discussion by The-Modding-Calculator** [CrossCraft/CrossCraft-SurvivalTest#5] [Q&A]
I wasn't sure where the right place to ask this was so sorry if I'm asking in the wrong place.
When I try to play the CrossCraft Survival Test on the PS Vita I crash after pressing Singleplayer on the main menu screen. After pressing the button, a pop appears stating an error has occurred and show error code C2-12828-1 and states it has created core file. I have attached the core file it has created in case it may be of any help.
[psp2core-1663616656-0x00025a203d-eboot.bin.psp2dmp.zip](https://github.com/CrossCraft/CrossCraft-SurvivalTest/files/9601978/psp2core-1663616656-0x00025a203d-eboot.bin.psp2dmp.zip)

I have tried both the latest release and the latest build from the action tab (This one at the time of writing: https://github.com/CrossCraft/CrossCraft-SurvivalTest/actions/runs/3003049103)

My vita is on firmware 3.60 with iTLS-Enso installed.
I have also installed libshacccg.suprx with the instructions linked in the realise page for Survival Test 3: Combat! 

Any help is appreciated, thanks
The Calculator

**Comment by IridescentRose** [comment]
Known issue

**Reply by The-Modding-Calculator** [reply]
Ok thanks
I might try some of the older builds and see if I can find one that'll work for now until this issue get's fixed and realised in a new build.
This is a pretty neat project I'm looking forward to seeing it's progress
